samples/analyze.v1p3beta1.js

```diff
@@ -0,0 +1,362 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function detectPerson(path) {
+  //[START video_detect_person_beta]
+  // Imports the Google Cloud Video Intelligence library + Node's fs library
+  const Video = require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+  // Creates a client
+  const video = new Video.VideoIntelligenceServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const path = 'Local file to analyze, e.g. ./my-file.mp4';
+
+  // Reads a local video file and converts it to base64
+  const file = fs.readFileSync(path);
+  const inputContent = file.toString('base64');
+
+  const request = {
+    inputContent: inputContent,
+    features: ['PERSON_DETECTION'],
+    videoContext: {
+      personDetectionConfig: {
+        // Must set includeBoundingBoxes to true to get poses and attributes.
+        includeBoundingBoxes: true,
+        includePoseLandmarks: true,
+        includeAttributes: true,
+      },
+    },
+  };
+  // Detects people in a video
+  const [operation] = await video.annotateVideo(request);
+  const results = await operation.promise();
+  console.log('Waiting for operation to complete...');
+
+  // Gets annotations for video
```

Comment:
Add a note that we get the first result because only 1 video is processed.

Reply:
Done in [this pull request](https://github.com/googleapis/nodejs-video-intelligence/pull/369).